content/nginx-one/glossary.md

```diff
@@ -15,9 +15,11 @@ This glossary defines terms used in the F5 NGINX One Console and F5 Distributed
 | Term        | Definition |
 |-------------|-------------|
 | **Config Sync Group** | A group of NGINX systems (or instances) with identical configurations. They may also share the same certificates. However, the instances in a Config Sync Group could belong to different systems and even different clusters. For more information, see this explanation of [Important considerations]({{< ref "/nginx-one/how-to/config-sync-groups/manage-config-sync-groups.md#important-considerations" >}}) |
+| **Control Plane** | The control plane is the part of a network architecture that manages and controls the flow or data or traffic (the Data Plane). It is responsible for system-level tasks such as routing and traffic management. |
```

Comment:
Actually, I'm going to ask for a developer review in an internal Slack room

Reply:
the definition was provided by the Agent team

Reply:
resolving. unless there's some real objection, this shouldn't stop you.